### PR TITLE
feat: add custom error page

### PR DESCRIPTION
**Pull request by IgboPharaoh**
This pr adds a custom error page to be displayed when the user navigates to a non-existent page

Screenshot 2024-07-25 at 16 34 39

### CODE REVIEW

**Review by Emmanuel-Develops — APPROVED** [review]
Looks good.
ACK!

File: src/app/not-found.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import Link from "next/link";
+
+export default function notfound() {
```
Comment (Emmanuel-Develops) [inline src/app/not-found.tsx:3]: One nit: any reason for not using Pascal case?
i.e:
```suggestion
export default function NotFound() {
```

Comment (IgboPharaoh) [reply src/app/not-found.tsx:3]: I'll make necessary changes

**Comment by Emmanuel-Develops** [comment]
please update branch @IgboPharaoh so I can merge

**Comment by IgboPharaoh** [comment]
> please update branch @IgboPharaoh so I can merge

Done

**Pull request merged**